src/main/java/org/sunbird/workflow/controller/WorkFlowController.java

```diff
@@ -122,4 +122,10 @@ public ResponseEntity<SBApiResponse> getBulkUpdateStatus(@RequestHeader(Constant
 	public ResponseEntity<?> downloadBulkuplodFile(@PathVariable("fileName") String fileName) {
 		return workflowService.downloadBulkUploadFile(fileName);
 	}
+
+	@PostMapping("profile/updaterequest/withdraw")
```

Comment:
why do you need a new API ?
the transition API -- should handle all type of state changes... 
UI (learner portal  & admin portals) should call the transition API and provide the action as required -- this should work -- just simply enhance the states...